controllers/node_module_mapper_test.go

```diff
@@ -0,0 +1,100 @@
+package controllers_test
+
+import (
+	"context"
+
+	"github.com/go-logr/logr"
+	"github.com/golang/mock/gomock"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/qbarrand/oot-operator/api/v1alpha1"
+	ootov1alpha1 "github.com/qbarrand/oot-operator/api/v1alpha1"
+	"github.com/qbarrand/oot-operator/internal/client"
+	"github.com/qbarrand/oot-operator/internal/filter"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+var _ = Describe("FindModulesForNode", func() {
+	var ctrl *gomock.Controller
+	BeforeEach(func() {
+		ctrl = gomock.NewController(GinkgoT())
```

Comment:
You may want to declare `client` here and initialize it in `BeforeEach` as well.